aws_lambda_powertools/utilities/parameters/dynamodb.py

```diff
@@ -183,7 +178,7 @@ def _get(self, name: str, **sdk_options) -> str:
         # Explicit arguments will take precedence over keyword arguments
         sdk_options["Key"] = {self.key_attr: name}
 
-        return self.table.get_item(**sdk_options)["Item"][self.value_attr]
+        return str(self.table.get_item(**sdk_options)["Item"][self.value_attr])
```

Comment:
why always str? dynamodb can store all sorts of data types

Reply:
@risenberg-cyberark  - this is typing related, the outer type is excepting a str.

Reply:
I'd agree with @risenberg-cyberark and this points out that we might have incorrect return typing annotations, and that tests aren't covering negatives

Reply:
@heitorlessa ok then in some cases we expect a `str`, so we should maybe add some type checking there.